File: src/Web/SWARM.PuP.Web/Code/Models/PuPTag.cs
using MongoDB.Bson;

namespace SWARM.PuP.Web.Models
{
    public class PuPTag
    {
        private string _id;

        public PuPTag()
        {
        }

        public PuPTag(string key, string value)
        {
            this.Key = key;
            this.Value = value;
        }

        public string Id
        {
            get { return _id ?? (_id = ObjectId.GenerateNewId().ToString()); }
            set { _id = value; }
        }

        public string Key { get; set; }

        public string Value { get; set; }
    }
}

File: src/Android/app/src/main/java/com/swarmnyc/pup/module/models/PuPTag.java
package com.swarmnyc.pup.module.models;

public class PuPTag {
    String key;
    String value;

    public PuPTag()
    {
    }

    public PuPTag( final String key, final String value )
    {
        this.key = key;
        this.value = value;
    }

    public String getKey() {
        return key;
    }

    public String getValue() {
        return value;
    }
}
